telemetry/telemetry_record.py

```diff
@@ -1,25 +1,80 @@
-"""Record WebPageReplay
+"""Record WebPageReplay, modify, and record page load times
 
 Records running a set of target urls on the remote tablet, modifies cacheable
-objects, then re-runs to see page load time differences.
+objects, then re-runs to see page load time differences. Writes 1 har file for
+each url and modified url in page_load_time/data/hars/
 
-Usage: sudo python record_wpr.py
+Usage: sudo python telemetry_record.py -t 2
 
 Notes:
     Assumes - telemetry is in CHROMIUM_SRC/tools/telemetry/telemetry
             - page_sets is in CHROMIUM_SRC/tools/perf/page_sets
             - modify_wpr_delays.py is in CHROMIUM_SRC/third_party/webpagereplay
+            - url data set is in PLT_SRC/telemetry/targer_urls, line delimited
 
-TODO: Measure page load time before and after modifying
-TODO: Make sure to only get the most recent wpr file when moving / modifying
-TODO: Clean up path dependencies, move them to the top
+Workflow:
+    1. Clean up old session data
+    2. Read target_urls
+    3. Write a page_set for each target url and its modified version to
+       CHROMIUM_SRC/tools/perf/page_sets
+    4. Creates *.json files to point to archive files for modified urls
+       stored in CHROMIUM_SRC/tools/perf/page_sets/data as we only record
+       unmodified urls
+    5. Record wpr for each url and its modified version, save to
+       CHROMIUM_SRC/tools/perf/page_sets/data/
+    6. Move wpr files to PLT_SRC/telemetry/data/wpr_source/ for processing
+    7. Modify wpr and assume a perfect cache, save to
+       PLT_SRC/telemetry/data/wpr_source/
+    8. Copy local wpr files to benchmark, located at
+       CHROMIUM_SRC/tools/perf/page_set/data
+    For each trial:
+    9. Write CHROMIUM_SRC/tools/perf/benchmarks/telemetryBenchmarks.py, the
+       benchmark file for each url and modified url. We have 1 class for each
+       benchmark to run
+    10. Run telemetryBenchmarks.PageCyclerUrl{url index}{_pc}, save data for
```

Comment:
@colin-scott I pushed the most recent version of telemetry_record.py. It currently works as is, but does *not* have the --user_server_delays flag yet.